include/private/rdyconfigimpl.h

```diff
@@ -64,10 +64,14 @@ typedef struct {
   RDyFlowSource      source;
 } RDyPhysicsFlow;
 
+typedef struct {
+  PetscInt num_classes;  // number of sediment classes
```

Comment:
Do we want an `enabled` field here, or does `num_classes = 0` imply that sediment dynamics is turned off?

Reply:
`num_classes = 0` implies SD is off. Let's use `num_classes` because we will use it to extend SD for the case when multiple sediment classes are used in a simulation.